engine-nextgen/src/main/kotlin/graphql/nadel/enginekt/transform/query/NadelFieldToServiceRouter.kt

```diff
@@ -0,0 +1,58 @@
+package graphql.nadel.enginekt.transform.query
+
+import graphql.nadel.Service
+import graphql.nadel.enginekt.blueprint.NadelOverallExecutionBlueprint
+import graphql.nadel.enginekt.util.copyWithChildren
+import graphql.nadel.enginekt.util.makeFieldCoordinates
+import graphql.nadel.schema.NadelDirectives
+import graphql.normalized.NormalizedField
+import graphql.normalized.NormalizedQuery
+import graphql.schema.GraphQLSchema
+
+class NadelFieldToServiceRouter {
```

Comment:
Router is bit much: how about simply `NadelFieldToService` or just `FieldToService`